Tests/Marketplace/upload_packs.py

```diff
@@ -407,53 +407,62 @@ def create_corepacks_config(storage_bucket: Any, build_number: str, index_folder
 
     """
     required_core_packs = GCPConfig.get_core_packs(marketplace)
-    core_packs_public_urls = []
-    bucket_core_packs = set()
-    for pack in os.scandir(index_folder_path):
-        if pack.is_dir() and pack.name in required_core_packs:
-            pack_metadata_path = os.path.join(index_folder_path, pack.name, Pack.METADATA)
-
-            if not os.path.exists(pack_metadata_path):
-                logging.critical(f"{pack.name} pack {Pack.METADATA} is missing in {GCPConfig.INDEX_NAME}")
-                sys.exit(1)
-
-            with open(pack_metadata_path, 'r') as metadata_file:
-                metadata = json.load(metadata_file)
+    corepacks_files_names = [GCPConfig.CORE_PACK_FILE_NAME]
+    corepacks_files_names.extend(GCPConfig.get_core_packs_unlocked_files(marketplace))
+    for corepacks_file in corepacks_files_names:
+        logging.info(f"Creating corepacks file {corepacks_file}.")
+        core_packs_public_urls = []
+        bucket_core_packs = set()
+        for pack in os.scandir(index_folder_path):
+            if pack.is_dir() and pack.name in required_core_packs:
+                pack_metadata_path = os.path.join(index_folder_path, pack.name, Pack.METADATA)
+
+                if not os.path.exists(pack_metadata_path):
+                    logging.critical(f"{pack.name} pack {Pack.METADATA} is missing in {GCPConfig.INDEX_NAME}")
+                    sys.exit(1)
```

Comment:
Maybe do a `continue` inside this `if`, and accumulate the bad packs, and exit after the `for` loop, so the user could get the list of all bad packs, instead of fixing them one by one?